cilium/cmd/service_update.go

```diff
@@ -50,8 +51,9 @@ func init() {
 	serviceUpdateCmd.Flags().BoolVarP(&k8sLoadBalancer, "k8s-load-balancer", "", false, "Set service as a k8s LoadBalancer")
 	serviceUpdateCmd.Flags().BoolVarP(&k8sHostPort, "k8s-host-port", "", false, "Set service as a k8s HostPort")
 	serviceUpdateCmd.Flags().BoolVarP(&localRedirect, "local-redirect", "", false, "Set service as Local Redirect")
-	serviceUpdateCmd.Flags().StringVarP(&k8sTrafficPolicy, "k8s-traffic-policy", "", "Cluster", "Set service with k8s externalTrafficPolicy as {Local,Cluster}")
```

Comment:
Shouldn't this option be deprecated for one release before being deleted? I don't see it in a list of deprecated options in 1.12.

Reply:
Given this the `service update` cmdline is mainly used for local testing, we don't need compat imho.

Reply:
Should we mark these flags as hidden if they're not for users?